docs/trex_release-notes.md

```diff
@@ -11,6 +11,27 @@ layout: docs
 ----
 See also: [Known Issues]({{site.baseurl}}/docs/trex_known_issues.html)
 
+
+### Tableau 2022.3
```

Comment:
We are releasing this with 2022.4, not 2022.3.

Reply:
@johnDance  I thought we were announcing VizAPI v2 for the Tableau 2022.3 (release)? It works with the 1.9 library. Damodar did all the work earlier. 
It's fine with me if we wait until 2022.4 and 1.10 library to do the release.

Reply:
We just never announced it, or shared it in the slack channel for extensions.
There are 2 options:
1. Keep this 2022.3 and make another release note for 2022.4 and pagination
2. Add them both together into the 2022.4 / 1.10.0 release.

Reply:
I think 2 is the best option. The VizAPI update should be part of a some kind of announcement and promo for the Dashboard Extensions API. This way we can update the `createVizImageAsync` method in the API reference (assuming we can still do a doc update to the library for 1.10??).   I've also modified Stephen's vizImage sample to work with the v2 spec. We could include that as a v2 sample.   We should let Wes and the team know what we're thinking.

Reply:
Okay. Do you want to check this one in, and then have another update for pagination, or hold on this until pagination is ready?

Reply:
What's the best practice?  I just added the sample for VizAPI v2 that could use some eyes. 
If it's all OK, seems like we should keep this atomic and make a new commit for pagination and the 1.10 library. I'd adjust the release notes at that time to lump them altogether.

Reply:
Let's change 2022.3 to 2022.4, 1.9 to 1.10, and check in this change.